loader/log.c

```diff
@@ -120,6 +120,10 @@ void loader_log(const struct loader_instance *inst, VkFlags msg_type, int32_t ms
 
         if ((msg_type & VULKAN_LOADER_PERF_BIT) != 0) {
             type = VK_DEBUG_UTILS_MESSAGE_TYPE_PERFORMANCE_BIT_EXT;
+        } else if ((msg_type & VULKAN_LOADER_VALIDATION_BIT) != 0) {
+            // For loader logging, if it's a validation message, we still want to also keep the general flag as well
+            // so messages of type validation can still be triggered for general message callbacks.
+            type = VK_DEBUG_UTILS_MESSAGE_TYPE_GENERAL_BIT_EXT | VK_DEBUG_UTILS_MESSAGE_TYPE_VALIDATION_BIT_EXT;
```

Comment:
Would this mean that validation messages get reported as general? Just looking for clarification when/how this is applied.

Reply:
No, this only means that it will be reported both to the "GENERAL" and "VALIDATION" message buckets.  The error/warning/info level is separate from this.

Reply:
Oh I see, that makes more sense now.

Reply:
Only ones from the loader.  These messages have been reported as general up to this point.  I don't know if we want to remove that since the loader is so important.